### PR TITLE
Chore (Objects): Add text object definition

### PR DESCRIPTION
**Pull request by oguzhankoral**
## Description & motivation

I need a text object definition to render some texts on viewer with specklepy.

## Checklist:

- [ ] My pull request follows the guidelines in the [Contributing guide](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)?
- [ ] My pull request does not duplicate any other open [Pull Requests](../../pulls) for the same update/change?
- [ ] My commits are related to the pull request and do not amend unrelated code or documentation.
- [ ] My code follows a similar style to existing code.
- [ ] I have added appropriate tests.
- [ ] I have updated or added relevant documentation.

### CODE REVIEW

**Review by gjedlicska — COMMENTED** [review]
Why are there differences between this and the Sharp object?

https://github.com/specklesystems/speckle-sharp/blob/main/Objects/Objects/Other/Text.cs

**Comment by oguzhankoral** [comment]
> Why are there differences between this and the Sharp object?
> 
> https://github.com/specklesystems/speckle-sharp/blob/main/Objects/Objects/Other/Text.cs

Sharp Text Object mostly designed for Rhino and AutoCAD (since we do not have them in specklepy), that's why it has display value too. But current support of `Objects.Other.Text` object doesn't need to have `displayValue` or `richText` to render it on viewer. Should we add them as placeholders even if they are empty?

**Comment by gjedlicska** [comment]
Yes please, we already have a lot of differences between sharp and py objects. Its a lot better to strive for the same schema if we can

File: src/specklepy/objects/other.py [hunk 1]
```diff
@@ -71,6 +71,19 @@ class DisplayStyle(Base, speckle_type=OTHER + "DisplayStyle"):
...
+class Text(Base, speckle_type=OTHER + "Text"):
+    """
+    Text object to render it on viewer.
+    """
+    plane: Plane
+    value: str
+    height: float
+    rotation: float
+    displayStyle: DisplayStyle
+    displayValue: Optional[Polyline] = None
```
Comment (gjedlicska) [inline src/specklepy/objects/other.py:83]: this is a List[Polyline] in sharp

File: src/specklepy/objects/other.py [hunk 2]
```diff
@@ -71,6 +71,19 @@ class DisplayStyle(Base, speckle_type=OTHER + "DisplayStyle"):
...
 
+class Text(Base, speckle_type=OTHER + "Text"):
+    """
+    Text object to render it on viewer.
+    """
+    plane: Plane
+    value: str
+    height: float
+    rotation: float
+    displayStyle: DisplayStyle
```
Comment (gjedlicska) [inline src/specklepy/objects/other.py:82]: Sharp doesn't have this, why is this needed?

**Review by gjedlicska — CHANGES_REQUESTED** [review]

File: src/specklepy/objects/other.py (hunk 1 above)
Comment (oguzhankoral) [reply src/specklepy/objects/other.py:83]: you are right!

File: src/specklepy/objects/other.py (hunk 2 above)
Comment (oguzhankoral) [reply src/specklepy/objects/other.py:82]: Because text color retrieves from `DisplayStyle`'s color. Otherwise text cannot be render without `displayValue`. I believe sharp missing this one

**Review by gjedlicska — APPROVED** [review]
